workers/docker-compose.yml

```diff
@@ -1,94 +1,145 @@
-      AMQP_MANAGEMENT_PORT: 15672
-      AMQP_USERNAME: ${AMQP_USERNAME}
-      AMQP_PASSWORD: ${AMQP_PASSWORD}
-      AMQP_VHOST: ${AMQP_VHOST}
+      AMQP_QUEUE: "job_ftp"
+      AMQP_HOSTNAME: "${AMQP_HOSTNAME}"
+      AMQP_PORT: "${AMQP_PORT}"
+      AMQP_MANAGEMENT_PORT: "${AMQP_MANAGEMENT_PORT}"
+      AMQP_USERNAME: "${AMQP_USERNAME}"
+      AMQP_PASSWORD: "${AMQP_PASSWORD}"
+      AMQP_VHOST: "${AMQP_VHOST}"
+      AMQP_TLS: "${AMQP_TLS}"
+      RUST_LOG: "info"
+      BACKEND_PASSWORD: "${ROOT_PASSWORD}"
+      BACKEND_HOSTNAME: "http://${HOSTNAME}:${BACKEND_PORT}/api"
+      BACKEND_USERNAME: "${ROOT_EMAIL}"
+    networks:
+      - mediacloudai_global
+      - workers
 
-      RUST_LOG: "debug"
-#    deploy:
-#      restart_policy:
-#        condition: on-failure
-#        delay: 5s
-#        max_attempts: 3
-#        window: 30s
+  ffmpeg:
+    image: mediacloudai/py_ffmpeg_worker:latest
+    volumes:
+      - ${PWD}/data:/data
+    environment:
+      AMQP_QUEUE: "job_ffmpeg"
+      AMQP_HOSTNAME: "${AMQP_HOSTNAME}"
+      AMQP_PORT: "${AMQP_PORT}"
+      AMQP_MANAGEMENT_PORT: "${AMQP_MANAGEMENT_PORT}"
+      AMQP_USERNAME: "${AMQP_USERNAME}"
+      AMQP_PASSWORD: "${AMQP_PASSWORD}"
+      AMQP_VHOST: "${AMQP_VHOST}"
+      AMQP_TLS: "${AMQP_TLS}"
+      RUST_LOG: "info"
     networks:
-      - global
+      - mediacloudai_global
+      - workers
 
-  rs_http_worker:
-    image: mediacloudai/rs_http_worker:latest
+  ism:
+    image: mediacloudai/rs_manifest_worker:latest
     volumes:
       - ${PWD}/data:/data
     environment:
-      AMQP_HOSTNAME: "localhost"
-      AMQP_PORT: 5672
-      AMQP_MANAGEMENT_PORT: 15672
-      AMQP_USERNAME: ${AMQP_USERNAME}
-      AMQP_PASSWORD: ${AMQP_PASSWORD}
-      AMQP_VHOST: ${AMQP_VHOST}
+      AMQP_QUEUE: "job_ism_manifest"
+      AMQP_HOSTNAME: "${AMQP_HOSTNAME}"
+      AMQP_PORT: "${AMQP_PORT}"
+      AMQP_MANAGEMENT_PORT: "${AMQP_MANAGEMENT_PORT}"
+      AMQP_USERNAME: "${AMQP_USERNAME}"
+      AMQP_PASSWORD: "${AMQP_PASSWORD}"
+      AMQP_VHOST: "${AMQP_VHOST}"
+      AMQP_TLS: "${AMQP_TLS}"
+      RUST_LOG: "info"
+      MANIFEST_MODE: "ISM"
+    networks:
+      - mediacloudai_global
+      - workers
 
-      RUST_LOG: "debug"
-#    deploy:
-#      restart_policy:
-#        condition: on-failure
-#        delay: 5s
-#        max_attempts: 3
-#        window: 30s
+  dash:
+    image: mediacloudai/rs_manifest_worker:latest
+    volumes:
+      - ${PWD}/data:/data
+    environment:
+      AMQP_QUEUE: "job_dash_manifest"
+      AMQP_HOSTNAME: "${AMQP_HOSTNAME}"
+      AMQP_PORT: "${AMQP_PORT}"
+      AMQP_MANAGEMENT_PORT: "${AMQP_MANAGEMENT_PORT}"
+      AMQP_USERNAME: "${AMQP_USERNAME}"
+      AMQP_PASSWORD: "${AMQP_PASSWORD}"
+      AMQP_VHOST: "${AMQP_VHOST}"
+      AMQP_TLS: "${AMQP_TLS}"
+      RUST_LOG: "info"
+      MANIFEST_MODE: "DASH"
+    networks:
+      - mediacloudai_global
+      - workers
+
+  acs:
+    image: registry.ftven.net/media-cloud-ai/docker-acs-worker:1.0.0
```

Comment:
can we define ftv-workers as the image is private, people without access will still in error ?

Reply:
What do you mean by "define ftv-workers"?

Reply:
I create an issue to solve that.